src/js/utils/lazzzy.js

```diff
@@ -23,7 +23,7 @@ export default class Lazzzy {
 		}
 	}
 
-	static loadImage(imageSelector, typeImage) {
+	loadImage(imageSelector, typeImage) {
```

Comment:
__[ESLint]__

Error - Expected 'this' to be used by class method 'loadImage'. (class-methods-use-this)


<!-- Comment by [SideCI](https://sideci.com) -->